lib/bsd/pty.c

```diff
@@ -67,7 +67,7 @@ static void
 mkserver(void)
 {
 	int fd, i;
-	char * const argv[3]={"ptyfs", 0};
+	const char *argv[3]={"ptyfs", 0};
```

Comment:
Avoiding warnings in execve. In fact there's a big problem out there: usually all exec funcs should have char * const * in their arguments, but actually they are mixed, so I don't know what of both is the right choice.